client/ionic/src/App.tsx

```diff
@@ -37,30 +34,15 @@ import './theme/variables.css';
 const App: React.FC = () => (
   <IonApp>
     <IonReactRouter>
-      <IonTabs>
         <IonRouterOutlet>
-          <Route path="/tab1" component={Tab1} exact={true} />
-          <Route path="/tab2" component={Tab2} exact={true} />
+          <Route path="/about" component={About} exact={true} />
```

Comment:
noting that down the line, we may want to rename this page `Onboarding` or something like that. it's possible we have a separate about page in the app in the future. just an FYI here, no need to change

Reply:
Perhaps an issue for routing after we get the process flow diagram?